nixos/tests/libreswan.nix

```diff
@@ -100,7 +100,7 @@ in
           Starts eavesdropping on Alice and Bob
           """
           match = "src host alice and dst host bob"
-          eve.execute(f"tcpdump -i br0 -c 1 -Avv {match} >/tmp/log &")
+          eve.execute(f"tcpdump -i br0 -c 1 -Avv {match} >/tmp/log >&2 &")
```

Comment:
This change is wrong and broke the test entirely.
I don't know why you didn't test this properly (was this done in a hurry to unblock the channel?), but you could have at least pinged the maitnainers.

Reply:
Yes, both the commits and the merge were made in a hurry. I hope you didn't lose much time on this. The rest of the PR must have saved many others time.

Reply:
I understand, thank you. It's just that it's hard to keep up with everything happening in nixpkgs.
I lost some time, but that mostly on me for not noticing this commit sooner.